sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -31,17 +31,22 @@
                     />
                     <button
                         name="set_to_draft"
-                        states="expired,done"
+                        states="cancel,done"
                         string="To Draft"
                         class="btn-secondary"
                         type="object"
                     />
                     <button
                         name="action_cancel"
-                        states="open"
+                        states="draft,open"
```

Comment:
**Chore**: Since you added the `attrs` below, you can remove `states`, which is now unused and overwritten by `attrs`.